browser-test/src/program_deep_link.test.ts

```diff
@@ -5,7 +5,7 @@ import {
   loginAsAdmin,
   loginAsGuest,
   loginAsTestUser,
-  logout,
+  logout, resetContext,
```

Comment:
Please put resetContext on a new line. /bin/fmt will probably fix it automatically.